ghost/core/core/server/services/milestones/service.js

```diff
@@ -69,10 +72,33 @@ module.exports = {
     },
 
     /**
+     * @param {number} [customTimeout]
+     *
      * @returns {Promise<object>}
      */
-    async initAndRun() {
+    async initAndRun(customTimeout) {
```

Comment:
Is this new param only for testing? Perhaps the number should come from config, or we can just use the fake clock in the tests to advance by 4 days 🤷‍♂️

Reply:
Yes, this is just for testing purposes

Reply:
Added a note. The custom timeout won't last long in the codebase, as we'll soon move to properly schedule the job. I left it in there for now 😅